ckanext/publications_qld_theme/templates/package/read.html

```diff
@@ -2,7 +2,20 @@
 
 {% block meta %}
 {{ super() }}
+<meta name="DCTERMS.title" content="{{ pkg['title'] or pkg['name'] }}" />
+
+<meta name="DCTERMS.publisher" scheme="AGLSTERMS.AglsAgent" content="corporateName=The State of Queensland; jurisdiction=Queensland" />
 {% if pkg['organization'] and pkg['organization']['title'] %}
-  <meta name="DCTERMS.creator" content="c=AU; o=The State of Queensland; ou={{ pkg['organization']['title'] }}" scheme="AGLSTERMS.GOLD" />
+  <meta name="DCTERMS.creator" scheme="AGLSTERMS.GOLD" content="c=AU; o=The State of Queensland; ou={{ pkg['organization']['title'] }}" />
+{% endif %}
+<meta name="DCTERMS.created" content="{{ h.reformat_date_string(pkg['metadata_created']) }}" />
+{% if pkg['metadata_modified'] %}
+  <meta name="DCTERMS.modified" content="{{ h.reformat_date_string(pkg['metadata_modified']) }}" />
 {% endif %}
+<meta name="DCTERMS.description" content="{{ pkg['notes'] or pkg['title'] or pkg['name'] }}" />
```

Comment:
should we html escape description as it can contain html code etc.

Reply:
That's automatic.